alvr/dashboard/src/steamvr_launcher/mod.rs

```diff
@@ -107,6 +110,23 @@ fn unblock_alvr_driver_within_vrsettings(text: &str) -> Result<String> {
     Ok(serde_json::to_string_pretty(&settings)?)
 }
 
+pub fn get_default_steamvr_executable_path() -> String {
+    let steamvr_root_dir = alvr_server_io::steamvr_root_dir()
+        .map_err(|e| error!("Couldn't find OpenVR or SteamVR files. {e}"))
+        .unwrap_or_default();
```

Comment:
I think this will be fine, but I'm really not sure what the default path could be so this could end up behaving quite weird. You should use `?` instead of the `unwrap_or_default()` and having the function return an option. On the call site that would then look like `if let Some(path) = get...()  {...}`.